waku/api.go

```diff
@@ -442,6 +447,7 @@ type Message struct {
 	P2P       bool             `json:"bool,omitempty"`
 }
 
+// TODO why not remove? This has no usage.
 type messageOverride struct { // nolint: deadcode,unused
 	Sig     hexutil.Bytes
 	Payload hexutil.Bytes
```

Comment:
I feel this code just needs to be removed. I've checked it and there are no usages in the code base.